contracts/teller-network/Escrow.sol

```diff
@@ -25,22 +28,28 @@ contract Escrow is Pausable, MessageSigned {
     struct EscrowTransaction {
         address payable seller;
         address payable buyer;
-        uint amount;
+        uint tokenAmount;
         address token;
         uint expirationTime;
-        bool released;
-        bool canceled;
-        bool paid;
         uint rating;
+        uint256 tradeAmount;
+        TradeType tradeType;
+        EscrowStatus status;
     }
 
+    enum TradeType {FIAT, CRYPTO}
```

Comment:
Currently there is only 1 trade type right?

Reply:
I was thinking of using the tradeType and tradeAmount to store the information from this screen. Maybe we can use a string data type instead of those two fields and store the info we need directly on that string?  (to give an example, If i want to open a trade for 500USD, the field would have a string containing `FIAT:500:USD`, or if the trade is for 12500SNT,  `CRYPTO:12500:SNT`. 

![screenshot](https://user-images.githubusercontent.com/1106587/53432345-b5ed9700-39c8-11e9-961c-6003110dfea1.png)